INTITA/Networking/Services/Authorization.swift

```diff
@@ -23,6 +22,17 @@ enum HTTPType {
             return AuthorizationFailing()
         }
     }
+    
+    var editService: EditUserProtocol {
+        switch self {
+        case .real:
+            return EditUserReal.shared
+        case .mock:
+            return EditUserMock()
+        case .fail:
+            return EditUserFailing()
+        }
+    }
```

Comment:
I think it's unnecessary.

Reply:
It is better to add 
`func editUserInfo(newUser: EditingUser, completion: @escaping (Error?) -> Void) {`

Reply:
Done.